InstagramClone/InstagramClone/Profile/View/StoriesCell/ProfileStoriesTableViewCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  StoriesTableViewCell.swift
+//  InstagramClone
+//
+//  Created by Admin on 23.07.2021.
+//
+
+import UIKit
+
+final class ProfileStoriesTableViewCell: UITableViewCell {
+    
+    //MARK: - IBOutlets
+    
+    @IBOutlet weak var storiesScrollView: UICollectionView!
+    
+    //MARK: - UITableViewCell
+    
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+        selectionStyle = .none
+    }
+    
+    //MARK: - Private Properties
+    
+    private let identifier = "SingleStoryCollectionViewCell"
+    private let images: [UIImage] = {
+        var images = [UIImage]()
+        for image in 1...22 {
+            images.append(UIImage(named: "\(image)")!)
```

Comment:
```suggestion
            images.append(UIImage(named: "\(image)"))
```
Это уже 20 приседаний друг, ты че за такое в продет знаешь че сделают?)